src/validation.cpp

```diff
@@ -4227,6 +4227,14 @@ bool static LoadBlockIndexDB(ChainstateManager& chainman, const CChainParams& ch
     return true;
 }
 
+void CChainState::LoadMempool(const ArgsManager& args)
+{
+    if (args.GetArg("-persistmempool", DEFAULT_PERSIST_MEMPOOL)) {
+        ::LoadMempool(m_mempool);
```

Comment:
This is the only call to the standalone `LoadMempool` function, so --in a follow-up-- its body could be moved here (or as a CTxMempool method) ?

Reply:
There is a fuzzer that uses the standalone function. (Pull might not be merged yet, though)

Reply:
#19259